### DISCUSSION TITLE
Documentation does not work - 404 links for lots of pages and search results are all returning 404

### DISCUSSION THREAD

**Discussion by mikebski** [fastapi/fastapi#11686] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import FastAPI

from db import get_apps, get_events

app = FastAPI()


@app.get("/")
async def root():
    return {"status": "OK", "apps": list([x['name'] for x in get_apps()])}

@app.get("/apps")
async def apps():
    return get_apps()
```

### Description

Click on any link on the docs site https://fastapi.tiangolo.com/

Watch the 404 pages show up

Searching also gives 404 errors

I'm new and looking for a simple post request doc, and cannot find one.

### Operating System

macOS

### Operating System Details

None needed, using FF on Mac

### FastAPI Version

0.111.0

### Pydantic Version

2.7.3

### Python Version

3.12.2

**Comment by YuriiMotov** [comment]
Can confirm.
Some pages show just white screen. As an example: https://fastapi.tiangolo.com/advanced/

**Comment by AlexanderMishutkin** [comment]
Only temporary solution I can suggest for now is to use Web Archive https://web.archive.org/web/20240324095137/https://fastapi.tiangolo.com/tutorial/
